settings.json.template

```diff
@@ -409,6 +409,25 @@
   "indentationOnNewLine": false,
   */
 
+  /*
+   * Import Rate limiting, File size and If a session is required to import
+   * 
+   * importRateLimiting defaults at 10 requests per IP per 90 seconds.  After this it's rejected, value is in mS.  See express-rate-limit for more options
+   * requireAuthorSessionToImport defaults to true meaning that an author must be on the pad to import.
+   * importMaxFileSize defaults to 50Mb meaning that a single imported file max size is 50Mb.  File size is specified in bytes.
+   * 
+   * Uncomment to customize.
+   */
+
+  /*
+  "importRateLimiting": {
+    "windowMs": 90000,
```

Comment:
I'd go with a seconds granularity. Too many digits for small benefit using ms.

Reply:
I use whatever the middleware uses.  https://www.npmjs.com/package/express-rate-limit  -- And I think we should stick with that so we can refer to their docs.

Reply:
Uhg, I still do not like it (end users/administrators are not required to know the internals of the library we are using, we do. And a human thinks in terms of seconds at best).

But I can also live with ms for sure. :)

Reply:
Kept `windowMs`.